src/operator/operator_common.h

```diff
@@ -414,6 +401,14 @@ void FCompExFallback(const nnvm::NodeAttrs& attrs,
           << ") == " << param << ".shape[0] (" << rsp.shape()[0] << ").";          \
   }
 
+template<typename xpu, int ndim, typename DType>
+inline mshadow::Tensor<xpu, ndim, DType> AllocateTempDataForSparseHandling(
```

Comment:
This is for alpha-only and will be removed.  It is intentionally named to not appear as a general-purpose function.